inginious/frontend/installer.py

```diff
@@ -376,20 +375,54 @@ def _build_container(self, name, folder):
         self._display_info("done.".format(name))
 
     def select_containers_to_build(self):
+        #If on a dev branch, download from github master branch (then manually rebuild if needed)
+        #If on an pip installed version, download with the correct tag
         if not self._ask_boolean("Build the default containers? This is highly recommended, and is required to build other containers.", True):
             self._display_info("Skipping container building.")
             return
 
+        # Get the local images
+        stock_images = []
+        try:
+            docker_connection = docker.from_env()
+            for image in docker_connection.images.list():
+                for tag in image.attrs["RepoTags"]:
+                    if re.match(r"^ingi/inginious-(c-[a-z]+|agent):v" + __version__, tag):
```

Comment:
After consideration, this is quite annoying because they are no functional requirement for the container tags to follow a particular name scheme. Moreover, even if we added this constraint, that would not allow third party containers to be pushed on the hub as the ``ingi/`` tag is reserved for the dev team.

Maybe the version check should only be done based on the docker agent version which is embedded as a label in the container. But there should be a way to compare with remote container data on the hub.

Reply:
+1

Reply:
Well, we changed our mind on this specific question and decided the scripts to only update the mandatory images (c-base and c-default). Third party containers should be handled manually anyway. And non-mandatory containers (such as java etc) will be correctly handled by inginious-install.